cafebabel/static/js/articles-tags.js

```diff
@@ -69,13 +69,11 @@ class Tags {
     return container
   }
   _createTag(tagValue, index) {
-    return `
-      <li>
+    return `<li>
         ${tagValue}
         <input name="tag-${++index}" list="tags" value=${tagValue} type="hidden">
```

Comment:
I suppose you need quotes nowhere but on the `value` attribute as it's the one that may have spaces or special chars 😅